src/lem/slot.rs

```diff
@@ -205,6 +227,8 @@ pub(crate) enum SlotType {
     Hash2,
     Hash3,
     Hash4,
+    Hiding,
+    IsDiffNeg,
```

Comment:
Could we call this `GreaterThan` or `LessThan`, whichever is appropriate based on the intended meaning?